website/content/en/docs/deployment/vanilla/guide-terraform.md

```diff
@@ -17,15 +17,39 @@ Specifially, you must:
 
 ## Deployment Steps
 
+### Directory
+
+Ensure you are in the `REPO_ROOT/deployments/vanilla/terraform` folder.
```

Comment:
can we add a sample cd and pwd command?

Ensure you are in the `REPO_ROOT/deployments/vanilla/terraform` folder.

If you are in repository's root folder, run
```
cd deployments/vanilla/terraform
pwd
```

Reply:
done